src/helpers/isUrl.ts

```diff
@@ -0,0 +1,5 @@
+export default function (url: string) {
+  return /^https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b(?:[-a-zA-Z0-9()@:%_+.~#?&/=]*)$/.test(
```

Comment:
It should also be invalid if the url has `/` in the end, because we add `/prove` in the code